styles.css

```diff
@@ -0,0 +1,457 @@
+  align-items: center;
+
+  .logo {
+    color: white;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    text-decoration: none;
+  }
+
+  .icon-menu {
+    #img-1 {
+      width: 18px;
+      height: 12px;
+      display: flex;
+      align-items: center;
+      flex: none;
+      order: 0;
+      flex-grow: 0;
+      text-decoration: none;
+    }
+  }
+}
+
+.main-content {
+  margin: 0 auto;
+  width: 375px;
+  background-color: #1c1a19;
+  background-image: url(images/Illustration.svg);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: 125%;
+
+  .section-1 {
+    height: 590px;
+    display: flex;
+
+    .article {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      width: 327px;
+      height: 124px;
+
+      .intro {
+        margin: 0;
+        padding: 120px 24px 0 24px;
+        font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 56px;
+        line-height: 64px;
+        letter-spacing: 0.37px;
+        color: #ff6b00;
+        display: flex;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+      }
+
+      .job-title {
+        margin-top: 12px;
+        padding: 0 24px 0 24px;
+        font-family: 'Roboto', Arial, Helvetica, sans-serif;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 24px;
+        line-height: 32px;
+        display: flex;
+        color: #fff5e1;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+      }
+
+      .job-description {
+        margin-top: -12px;
+        padding: 0 24px 0 24px;
+        font-family: 'Poppins', Arial, Helvetica, sans-serif;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 16px;
+        line-height: 24px;
+        color: #fff;
+        display: flex;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+      }
+
+      .links {
+        width: 160px;
+        margin: 0 24px;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-around;
+        padding: 0;
+        flex: none;
+        order: 3;
+        flex-grow: 0;
+
+        .socials {
+          list-style: none;
+        }
+      }
+    }
+  }
+
+  .section-2 {
+    display: flex;
+    margin: 0 auto;
+    width: 375px;
+    background-color: #fff;
+    margin-top: 100px;
+
+    .recent-article {
+      width: 327px;
+      margin: 0 auto;
+      display: flex;
+      flex-direction: column;
+    }
+
+    .recent-text {
+      font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 40px;
+      color: #091e42;
+    }
```

Comment:
- [ ] Please according to the requirement, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=pBzk0OysG1xz0qYa-0). Please ensure that your font color is `#1C1A19`

Reply:
> * [ ]  Please according to the requirement, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=pBzk0OysG1xz0qYa-0). Please ensure that your font color is `#1C1A19`

Hi @Alusp, the color of this text in the template is `#091e42` as used by the student.